<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCMA Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Folders</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Specify the folders that the PS Vita will access for each content type.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="whatsThis">
                <string>This is the location your Screenshots and Pictures are Saved to/Imported from.</string>
               </property>
               <property name="text">
                <string>Photo Folder</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLineEdit" name="photoPath">
                 <property name="toolTip">
                  <string>This is the location your Screenshots and Pictures are Saved to/Imported from.</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="photoBtn">
                 <property name="text">
                  <string>Browse...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="whatsThis">
                <string>This is the location your Videos are Saved to/Imported from.</string>
               </property>
               <property name="text">
                <string>Video Folder</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLineEdit" name="videoPath">
                 <property name="toolTip">
                  <string>This is the location your Videos are Saved to/Imported from.</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="videoBtn">
                 <property name="text">
                  <string>Browse...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="whatsThis">
                <string>This is the location your Music is Saved to/Imported from.</string>
               </property>
               <property name="text">
                <string>Music Folder</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLineEdit" name="musicPath">
                 <property name="toolTip">
                  <string>This is the location your Music is Saved to/Imported from.</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="musicBtn">
                 <property name="text">
                  <string>Browse...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="whatsThis">
                <string>This is the location your Games, Apps, Savegames, and System Backups are Saved to/Imported from.</string>
               </property>
               <property name="text">
                <string>Applications / Backups</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLineEdit" name="appPath">
                 <property name="toolTip">
                  <string>This is the location your Games, Apps, Savegames, and System Backups are Saved to/Imported from.</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="appBtn">
                 <property name="text">
                  <string>Browse...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="whatsThis">
                <string>This is the location your Software Updates and Browser Data is Saved to/Imported from.</string>
               </property>
               <property name="text">
                <string>Updates / Web content</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLineEdit" name="urlPath">
                 <property name="toolTip">
                  <string>This is the location your PS Vita system will read all the content that it tries to download.</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="urlBtn">
                 <property name="text">
                  <string>Browse...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Other</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Advanced settings&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="offlineCheck">
             <property name="text">
              <string>Offline Mode</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="metadataCheck">
             <property name="text">
              <string>Skip metadata extraction</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="usbCheck">
             <property name="text">
              <string>Disable USB monitoring</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="wifiCheck">
             <property name="text">
              <string>Disable Wi-Fi monitoring</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_3">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Update database automatically when files on the PC are changed</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Database backend</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="databaseSelect">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>In Memory</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>SQLite</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="photoSkipCheck">
             <property name="text">
              <string>Skip photo scanning</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="videoSkipCheck">
             <property name="text">
              <string>Skip video scanning</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="musicSkipCheck">
             <property name="text">
              <string>Skip music scanning</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
